server/src/main/java/io/druid/indexing/overlord/autoscaling/ResourceManagementSchedulerConfig.java

```diff
@@ -39,6 +42,11 @@
   @JsonProperty
   private DateTime originTime = new DateTime("2012-01-01T00:55:00.000Z");
 
+  public String getRegion()
```

Comment:
We should not introduce AWS-specific concerns in `ResourceManagementSchedulerConfig`. Autoscaling is implementation agnostic, and EC2 is only one possible implementation of auto-scaling. Anything specific to EC2 should stay in the EC2-specific auto-scaling classes.